packages/block-library/src/embed/test/index.native.js

```diff
@@ -143,6 +174,7 @@ beforeAll( () => {
 	// block is added to empty posts.
 	registerBlock( paragraph );
 	registerBlock( embed );
+	setDefaultBlockName( paragraph.name );
```

Comment:
The default block is already being set when registering the core blocks, however, we weren't setting it in the integration tests:
https://github.com/WordPress/gutenberg/blob/18768335e73d32a99e326f3eaa1250372a412b26/packages/block-library/src/index.native.js#L273

Although it's not necessary for rendering the core blocks, I found out that it was required by the autocomplete logic to display the slash inserter:
https://github.com/WordPress/gutenberg/blob/18768335e73d32a99e326f3eaa1250372a412b26/packages/block-editor/src/components/autocomplete/index.js#L37